### PR TITLE
First stab at paz-machine with drivers, as a built-in provision command

### PR DESCRIPTION
**Pull request by tomgco**

### CODE REVIEW

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -6,13 +6,26 @@
 	],
 	"Deps": [
 		{
-			"ImportPath": "github.com/coreos/fleet/version",
-			"Comment": "v0.3.2-1055-ga35ee29",
-			"Rev": "a35ee29f4f539d3d2296ac85c302f8b2e7d7a878"
+			"ImportPath": "code.google.com/p/goauth2/oauth",
```
Comment (sublimino) [inline Godeps/Godeps.json:9]: Using gcode from today is the first rung on a slippery slope :D Long live GH!

Comment (tomgco) [inline Godeps/Godeps.json:9]: God damn it, you are right.

Comment (tomgco) [inline Godeps/Godeps.json:9]: So the interface has changed, and the version used here is deprecated :/ going to take a while to figure out the new interface. _Looks at clock_ must sleep!

**Comment by lukebond** [comment]
Not sure if this is ready to be merged based on last comments?

**Comment by tomgco** [comment]
Its good to go as the source is committed however we will need to port
sometime in the future.

**Pull request merged**